tools/tests/csdl-16.1.xml

```diff
@@ -48,7 +48,9 @@
         <Key>
           <PropertyRef Name="ID" />
         </Key>
-        <Property Name="ID" Type="Edm.String" Nullable="false" />
+        <Property Name="ID" Type="Edm.String" Nullable="false">
+          <Annotation Term="Org.OData.Core.V1.ComputedDefaultValue" />
+        </Property>

```

Comment:
The `Supplier/ID` has a computed default value, but is still "required" during creation in the OpenAPI description:
https://github.com/oasis-tcs/odata-openapi/blob/e9d89610ce2d9e1f641ad88d75b2b8c6039a2134/tools/tests/csdl-16.1.openapi3.json#L2197-L2199

Can you check again, @VigneshSubramania91?